hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/state/SystemFileLoader.java

```diff
@@ -49,7 +51,11 @@ private Map<FileID, File> loadAll() {
     }
 
     private File load(int fileNum, Bytes contents) {
```

Comment:
We should add entries for each system file in `SystemEntity` then use that here. This avoids having the hardcoded numbers above and have a central way to trace these files in the codebase.

```suggestion
    private File load(SystemEntity systemFile, Bytes contents) {
```

Reply:
Addressed